packages/create-next-app/create-app.ts

```diff
@@ -158,7 +161,7 @@ export async function createApp({
         })
       }
     } catch (reason) {
-      throw new DownloadError(reason)
+      throw new DownloadError(reason + '')
```

Comment:
I think this might have been a bug. I don't think we're throwing a string anywhere.

Maybe it should check for `reason.message`?

```suggestion
      throw new DownloadError(reason && 'message' in reason ? reason.message : reason )
```